include/z64player.h

```diff
@@ -678,7 +678,7 @@ typedef struct WeaponInfo {
 #define PLAYER_STATE1_27 (1 << 27)
 #define PLAYER_STATE1_28 (1 << 28)
 #define PLAYER_STATE1_29 (1 << 29)
-#define PLAYER_STATE1_30 (1 << 30)
+#define PLAYER_STATE1_LOCK_ON_FORCED_RELEASE (1 << 30) // Lock-on was released automatically, for example by leaving the lock-on leash range
```

Comment:
the tense of the flag name vs the comment is confusing me, is this flag set in order to force release, or set after a force release happens?

Reply:
set after a forced release happens. `LOCK_ON_FORCE_RELEASED` is what youd want? that doesnt read quite right to me. `LOCK_ON_RELEASED_FORCEFULLY` (??? lol)

Reply:
lmao okay i actually don't hate `LOCK_ON_RELEASED_FORCEFULLY` at all. I think I get it now though. maybe `FORCED` isn't necessary? just `PLAYER_STATE1_LOCK_ON_RELEASED`? since part of the conditional to set it is simply `Attention_ShouldReleaseLockOn`

Reply:
its not set when you release the lock on yourself though (either pressing Z for switch, or letting go of Z for hold)

Reply:
I see.. alright, I don't have anything better than the original name then

Reply:
the attention function name can possibly add "force" for continuity, but yeah I think its necessary to convey that it wasnt the players choice.
also this flag doesnt mean much as far as I can tell. it gets checked for some things but in places where it doesnt make sense or is inconsequential